pkg/cmd/roachtest/tests/rust_helpers.go

```diff
@@ -0,0 +1,67 @@
+package tests
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"regexp"
+)
+
+var rustUnitTestOutputRegex = regexp.MustCompile(`(?P<type>.*) (?P<class>.*)::(?P<name>.*) .\.\.\ (?P<result>.*)`)
+
+func (r *ormTestsResults) parseRustUnitTestOutput(
+	input []byte, expectedFailures blocklist, ignoreList blocklist,
+) {
+	scanner := bufio.NewScanner(bytes.NewReader(input))
+	for scanner.Scan() {
+		x := scanner.Text()
+		match := rustUnitTestOutputRegex.FindStringSubmatch(x)
+		if match != nil {
+			groups := map[string]string{}
+			for i, name := range match {
+				groups[rustUnitTestOutputRegex.SubexpNames()[i]] = name
+			}
+			test := fmt.Sprintf("%s %s::%s", groups["type"], groups["class"], groups["name"])
+			skipped := groups["result"] == "skipped" || groups["result"] == "expected failure" || len(groups["result"]) > 4
```

Comment:
can you explain the `len(groups["result"]) > 4` thing?

Reply:
That was left over from testing. Removed it